modules/ppcp-wc-gateway/src/WCGatewayModule.php

```diff
@@ -284,7 +284,7 @@ static function ( $methods ) use ( $container ): array {
 					$methods[] = $container->get( 'wcgateway.credit-card-gateway' );
 				}
 
-				if ( 'DE' === $container->get( 'api.shop.country' ) ) {
+				if ( 'DE' === $container->get( 'api.shop.country' ) && 'EUR' === get_woocommerce_currency() ) {
```

Comment:
Maybe better to use https://github.com/woocommerce/woocommerce-paypal-payments/blob/1f3ee422ef108a45008791199989af98ccbc9e0b/modules/ppcp-api-client/services.php#L372

Reply:
Yup, much better, thanks.